ddtrace/opentelemetry/_span.py

```diff
@@ -16,19 +16,41 @@
 
 
 if TYPE_CHECKING:
+    from typing import Callable
     from typing import Mapping
     from typing import Optional
     from typing import Union
 
     from opentelemetry.util.types import Attributes
     from opentelemetry.util.types import AttributeValue
 
+    from ddtrace.internal.compat import NumericType
     from ddtrace.span import Span as DDSpan
 
 
 log = get_logger(__name__)
 
 
+def _ddmap(span, attribute, value):
+    # type: (DDSpan, str, Union[bytes, NumericType]) -> DDSpan
```

Comment:
Unless there's a good reason not to, I'd recommend making this an annotation rather than a comment.

Reply:
actually, a good reason: I want to avoid any other changes in this PR.